applications/luci-app-acme/htdocs/luci-static/resources/view/acme/acme.js

```diff
@@ -648,7 +648,7 @@ function _collectDdnsDomains() {
 		if (credentials.length > 0) {
 			ddnsDomains.push({
 				sectionId: ddnsService['.name'],
-				domains: [ddnsService['domain']],
+				domains: [ddnsService['domain'], '*.' + ddnsService['domain']],
```

Comment:
Why? I don't think we should be encouraging people to randomly issue wildcard certs...